### ISSUE TITLE
req.params are being lost

### ISSUE THREAD

**Issue by kurian86** [balderdashy/sails#2348]
When i execute sailsJS in production mode with forever all form input going after a file input are being lost from req.params.

Scenario
======
##### Forever start command: #####
forever start --uid "prod" -a -l logs/forever.log -o logs/out.log -e logs/err.log app.js --prod

##### Form #####
input type="text" name="text1"
input type="text" name="text2"
input type="file" name="file"
input type="text" name="text3"
input type="checkbox" name="checkbox1"

##### After Submit #####
req.params.all() just contains text1 and text2

If i move file to bottom form req.params.all() have all input, in dev mode with "sails lift" all work fine.

**Comment by CWyrtzen** [comment]
Any update on this?

**Comment by sailsbot** [comment]
Thanks for posting, @kurian86.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
